manifests/README.md

```diff
@@ -17,6 +17,9 @@ Four sets of installation manifests are provided:
   external clusters. Will still be possible to deploy to the same cluster (kubernetes.svc.default)
   with inputted credentials (i.e. `argocd cluster add <CONTEXT> --in-cluster --namespace <YOUR NAMESPACE>`).
 
+> Note: Argo CD CRDs are not included into [namespace-install.yaml](namespace-install.yaml) and have to be installed
+> separately.
```

Comment:
Can we give instructions on *how* to install the CRDs manually? E.g. where to find them in the code base?

Reply:
good point. added